spring-integration-dsl-groovy-core/src/main/groovy/org/springframework/integration/dsl/groovy/ClosureInvokingComponents.groovy

```diff
@@ -215,6 +216,15 @@ class ClosureInvokingChannelInterceptor implements org.springframework.messaging
 		result as Message
 	}
 
+	void afterSendCompletion(Message<?> message, MessageChannel channel, boolean sent, Exception ex) {
+		interceptor.afterSendCompletion
```

Comment:
Sorry, I forgot a bit Groovy or just haven't used this technique before, hence I don't understand how it works.
I see other methods here use these `Closure`s like `interceptClosure.call(....)`.
Don't we miss something similar here?
Shouldn't we have a test-case on the matter?